examples/tutorial/napa-runner/README.md

```diff
@@ -0,0 +1,14 @@
+# Napa runner
+Napa runner is a tool to run JavaScript with napa capability. It is implemented in C++ using napa API.
```

Comment:
Napa runner is an example to embed Napa.JS into a C++ program, it simply runs a JavaScript with all Napa capability without Node dependency from command line.